glomex-dialog.js

```diff
@@ -549,39 +602,40 @@ class GlomexDialogElement extends window.HTMLElement {
         });
       } else if (newValue === 'inline') {
         const goToInline = () => {
+          // somehow this avoids CLS when switching between
+          // position "fixed" => "absolute"
+          dialogContent.style.display = 'grid';
```

Comment:
haxx :)

Reply:
I still question how CLS measurement is implemented in Chrome anyways. So whatever works.